package.json

```diff
@@ -5,15 +5,25 @@
   "homepage": "https://github.com/unifiedjs/unified-language-server",
   "repository": "unifiedjs/unified-language-server",
   "bugs": "https://github.com/unifiedjs/unified-language-serve/issues",
-  "main": "src/index.js",
+  "main": "index.js",
   "author": "aecepoglu",
   "license": "MIT",
   "sideEffects": false,
-  "bin": "unified-language-server",
+  "type": "module",
+  "exports": {
+    ".": "./index.js"
```

Comment:
needed, or preference?

Reply:
Technically not necessary, but it marks other files as private.